app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -68,6 +67,11 @@ public function index(Request $request)
                 $query = $query->where('signups.northstar_id', '=', $filters['northstar_id']);
             }
 
+            // Only return Posts for the given status (if there is one)
+            if (array_has($filters, 'status')) {
+                $query = $query->where('status', '=', $filters['status']);
```

Comment:
ooohhhh good call! I don't think we need this at the moment but in case we need to in the future, should I add this now?